### PR TITLE
[MM-15413] Updated Base DN text to the proposed text

### PR DESCRIPTION
**Pull request by hahmadia**
#### Summary
This ticket updates the description of the Base DN description in the admin console

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-15413

#### Current
image

#### Fix
image

### CODE REVIEW

File: i18n/ko.json [hunk 1]
```diff
@@ -670,7 +670,7 @@
   "admin.jobTable.statusInProgress": "진행 중",
   "admin.jobTable.statusPending": " 보내는 중",
   "admin.jobTable.statusSuccess": "성공",
-  "admin.ldap.baseDesc": "The Base DN is the Distinguished Name of the location where Mattermost should start its search for users in the LDAP tree.",
+  "admin.ldap.baseDesc": "The Base DN is the Distinguished Name of the location where Mattermost should start its search for user and group objects in the AD/LDAP tree.",
```
Comment (deanwhillier) [inline i18n/ko.json]: Only the original English translations need to be provided for the Webapp repo, the rest should automatically get updated by https://translate.mattermost.com/. 🙂

So, no need to include this in the PR.

Comment (deanwhillier) [reply i18n/ko.json]: That said, I do see the discrepancy between the en.json and ko.json files before this PR. @enahum, should we be concerned that this entry in en.json and ko.json were different prior to this PR?

Comment (enahum) [reply i18n/ko.json]: that one will get included automatically if it does not get translated in the translation server. the only one thing to do in this PR is remove the changes in this file.

Comment (hahmadia) [reply i18n/ko.json]: Thanks for pointing this out! Done! 😄

**Pull request merged**